README.md

```diff
@@ -1,63 +1,58 @@
-# My Portfolio
-
-> This is a portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
-
-![screenshot](images/screenshot.png)
-
-
-## Built With
-
-- Html
-- CSS
-
-
-## Getting Started
-
-
-To get a local copy up and running follow these simple example steps
-
-### Prerequisites
-Github Account 
-
-### Setup
-Set up and configure git on your system.
-
-### Install
-Clone this repository
-
-
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@ifnotlily](https://github.com/ifnotlily)
-- Twitter: [@ifnotlily](https://twitter.com/ifnotlily)
-- LinkedIn: [lilian-francis](https://linkedin.com/in/lilian-francis)
-
-👤 **Author2**
-
-- GitHub: [@Wuyep Abdul](https://www.github.com/wuyepabdul)
-
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+# My Portfolio
+
+> This is a responsive portfolio project by microverse to test the use of Flexbox for Positioning and also grids, images and Background, 
+
+![screenshot](images/screenshot.png)
+
+
+## Built With
+

```

Comment:
Please align buttons with the above text section horizontally. See the 
![project2](https://user-images.githubusercontent.com/35504149/129944438-64cd0fbc-34b6-4f8b-aabf-676def838118.png)
below screenshot.